entrypoint.sh

```diff
@@ -12,6 +12,9 @@ set -ex
 
 cd phabricator
 
+rm src/__phutil_library_map__.php
```

Comment:
These two lines should have a comment, like "relink blah in case a developer changed it on disk during extension development".  Otherwise someone unfamiliar with extension development doing maintenance work on this script might not understand what the rm/relink behaviour is for.